loader.js

```diff
@@ -141,15 +141,16 @@ async function sendData(port, data) {
       console.log('Booting...');
       await sendCommand(port, prepareCommand(commands.BOOT), false);
     } else {
-      const partial = Buffer.from(data.buffer, offset, STATUS_UPDATE_AT);
+      var bytesToWrite = Math.min(size-offset, STATUS_UPDATE_AT);
```

Comment:
Better to use `const bytesToWrite`

Reply:
Also we can have spaces around `-`. Nitpicking here.

Opened #20, we have a prettier config that devs. are unable to use because it will not get installed by default.

You can globally install and run `prettier --write` though.